action.yml

```diff
@@ -72,45 +72,65 @@ runs:
       shell: bash
       run: |
         aws ecs update-service \
-          --health-check-grace-period-seconds 900 \
```

Comment:
@IamAbbey have tested this, unfortunately it seems that migrations count towards the grace period. So we need to think more how grace periods should work in general, in order to avoid running into failed deployments when migrations take a long time. This discussion is somewhat separate from the changes here though, so I think we can finish up the pull request here 👍